### DISCUSSION TITLE
Upload field. Files are not included in request.

### DISCUSSION THREAD

**Discussion by supercute** [Laravel-Backpack/community-forum#388] [Q&A (Help)] [answered]
I use spatie [laravel-medialibrary](https://spatie.be/docs/laravel-medialibrary/v10/introduction) for update media.  On edit action i need only url of files. When i try to save entry, files are not included in request.

```php
CRUD::modifyField(DocumentType::INVOICE->value, [
     'value' => $document->getMedia('invoices')->first()?->getUrl(),
]);
```

How can i render only file URL and add save a files in the request?

**Comment by pxpm** [comment]
Hey @supercute I have amazing news for you (or anyone using spatie media library):

![image](https://user-images.githubusercontent.com/7188159/217229489-ccaa623c-5465-459f-a0a6-240052de5e16.png)

It should be out during this/next week. 🤞 

Cheers

**Comment by jcastroa87** [comment, marked as answer]
Hi, here is finally the implementation you can check our [docs](https://backpackforlaravel.com/docs/6.x/crud-uploaders#uploaders-for-spatie-medialibrary)

Cheers.
